roles/download/defaults/main/checksums.yml

```diff
@@ -60,9 +80,13 @@ kubelet_checksums:
     v1.25.2: 995f885543fa61a08bd4f1008ba6d7417a1c45bd2a8e0f70c67a83e53b46eea5
     v1.25.1: 6fe430ad91e1ed50cf5cc396aa204fda0889c36b8a3b84619d633cd9a6a146e2
     v1.25.0: ad45ac3216aa186648fd034dec30a00c1a2d2d1187cab8aae21aa441a13b4faa
+    v1.24.15: 547147c8dd4096d975bb2669f0be500fff6f8fdd0848d25dace3d278ad1e3b88
```

Comment:
Dunno if you are not done yet but there's still a bunch of 1.24 version in there 🙈

Reply:
The number of versions included in the `checksums.yml` file is controlled by the [third argument of the get_version function](https://github.com/kubernetes-sigs/kubespray/blob/master/scripts/download_hash.sh#L16). For `kubeadm` and `kubelet`, the value is set to `25`. Should I need to change it to `23`?

Reply:
Ohhh I see I wasn't fully aware it worked like that sorry. So I think this should be more a minimum version for crio / core kubernetes versions than "last x versions". But IMO this could be done later on (or not as you wish), I wouldn't want to block this PR that was already left hanging for too long :(.